openmrs/apps/clinical/dashboard.json

```diff
@@ -8,9 +8,9 @@
                 "translationKey":"DASHBOARD_TITLE_PATIENT_INFORMATION_KEY",
                 "type":"patientInformation",
                 "displayOrder":0,
-                "ageLimit":100,
-                "patientAttributes":[  
-
+                "ageLimit":12,
+                "showDOB":false,
```

Comment:
"showDOB": false   stops the date of birth appearing again below

ageLimit is the age limit at which we stop displaying Months and Days after the Years